src/index.html

```diff
@@ -19,7 +13,9 @@
   <body>
     <div id="app" class="app-wrapper">
       <div className="loading-screen">
-        <div className="u-loading-spinner u-loading-spinner--light" style="
+        <div 
+        className="u-loading-spinner u-loading-spinner--light" 
+        style="
```

Comment:
This code change is also an example of an unrelated change that should be cleaned up/removed before submitting a pull request.